src/arraytraits.rs

```diff
@@ -47,6 +47,7 @@ where
 {
     type Output = S::Elem;
     #[inline]
+    #[track_caller]
     fn index(&self, index: I) -> &S::Elem {
```

Comment:
It doesn't work on trait methods I thought?

Reply:
https://doc.rust-lang.org/reference/attributes/codegen.html Looks like it does, but I should just put it on the trait definition not implementations

Reply:
Ah but Index and IndexMut are in the standard library... And already tracked there! I'll remove :+1: